### ISSUE TITLE
Add: Multi-Select elements

### ISSUE THREAD

**Issue by scottaohara** [scottaohara/a11y_styled_form_controls#48]
Fresh start for #7 

Add examples of styling a multi-`select` element.

- [ ] Multiple options viewable at once (max height / scrollable)
- [ ] Single option viewable at once
- [ ] Document component

Add other ideas as comments...

**Comment by paulshryock** [comment]
Hey! I was wondering if you still have plans to make this? I've been working on a multi-select element with reasonable styles and some progressive enhancements, but I don't love what I have. Curious to see how you'd approach this UI pattern.

**Comment by scottaohara** [comment]
hey @paulshryock 

no plans at this point.  TBH, at this point I'd be more inclined to wait for additional work to be done on the customizable `select` in HTML, rather than just minimal styling for what can be currently achieved with the element - or duplicating the ARIA pattern for a multi-select listbox.

Sorry, I know that's no helpful _right now_, but good luck with what you're working on!

**Comment by paulshryock** [comment]
Yeah, I've been looking at customizable `select`, but it seems that widespread support is still a ways off, and it doesn't include support for `multiple`.

> There isn't currently any support specified for the `multiple` attribute on customizable `<select>` elements, but this will be worked on in the future.
> -- https://developer.mozilla.org/en-US/docs/Learn_web_development/Extensions/Forms/Customizable_select#customizing_other_classic_select_features

[This ARIA Listbox pattern](https://www.w3.org/WAI/ARIA/apg/patterns/listbox/examples/listbox-rearrangeable/#ex2_label) probably has everything I need.

Thanks for the reply and suggestions.
